Directory.Build.props

```diff
@@ -23,6 +23,16 @@
   <PropertyGroup Condition="'$(Configuration)'=='Release'">
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
   </PropertyGroup>
+  <PropertyGroup Condition=" $(TargetFramework.StartsWith('net5')) Or $(TargetFramework.StartsWith('net6')) Or $(TargetFramework.StartsWith('net7')) ">
```

Comment:
I actually think all this belongs in the Directory.Build.targets file since it refers to the TargetFramework which is defined per project file. Thoughts?

Reply:
Yeah, seems like leftovers from my experiments :)
The difference is that props file imported before the project and targets is imported after.
I’ll check if I can put them together in one file.

Reply:
I used Net5OrAbove in csproj, but now I don't, so moving to targets.